xdem/coreg/base.py

```diff
@@ -826,21 +1181,25 @@ def fit(
         if self._meta["subsample"] != 1:
             self._meta["random_state"] = random_state
 
-        # Pre-process the inputs, by reprojecting and subsampling
-        ref_dem, tba_dem, inlier_mask, transform, crs = _preprocess_coreg_raster_input(
-            reference_dem=reference_dem,
-            dem_to_be_aligned=dem_to_be_aligned,
+        # TODO: Add preproc for points too
```

Comment:
do you plan to address those in this PR?

Reply:
This is done, I forgot to remove the TODOs, good catch!